### DISCUSSION TITLE
1年間の実績をシェアしたい

### DISCUSSION THREAD

**Discussion by duri0214** [increments/qiita-discussions#145] [要望・アイデア・不具合]
たしか1年に書いた記事数とかいいねの数とかがメールで送られてきたような記憶がありますが、それをフェイスブックとかでシェアしたいです（たしかそれができなくてスクショで貼りました）

なにかこう例えば企業に見せることもできるようなデザインだと嬉しいですね

**Comment by kskwtnk** [comment]
@duri0214 
ご意見ありがとうございます！

我々としても何かしらシェアがしやすいような形式にしたいと考えております！
まだ全然ちゃんと決まっていないのですが、例えば以下のような……

- サマリー画像を出力できる
- Web上にあり、URLだけでアクセスできる
- OGP画像がサマリーになっている

去年のものよりも充実させてシェアもしやすくする予定ですので、今年のレポートもまた楽しみにしていてください！

**Reply by P-SiZK** [reply]
@duri0214 
大変お待たせしました :bow: 

一年間の実績を見ることができる「ユーザー振り返りレポート」をシェアできるようになりました！

詳しい内容は[Qiita Blog](https://blog.qiita.com/yearly-summary-2024/)をご覧ください！
